packages/components/package.json

```diff
@@ -44,13 +44,13 @@
         "@storybook/core": "^6.5.10",
         "@storybook/manager-webpack5": "^6.5.10",
         "@storybook/react": "^6.5.10",
-        "@types/react": "^17.0.48",
+        "@types/react": "^18.0.21",
         "@types/react-date-range": "^1.4.4",
         "@types/react-truncate": "^2.3.4",
         "@types/styled-components": "^5.1.26",
         "jest": "^26.6.3",
-        "react": "17.0.2",
-        "react-dom": "17.0.2",
+        "react": "^18.2.0",
```

Comment:
nit: It doesn't matter since we have react in root resolution field but I would probably prefer not to use `^`. Same for `react-dom` and `@types/react`

Reply:
Sure, it was quite a mixed bag originally. 4a45a1a4f6456237d817a960b29152b05b3dbc29 requires exact versions of `react`, `react-dom`, `@types/react` and `@types/react-dom`.